### DISCUSSION TITLE
Does Tauri add any obfuscation to the binary during build?

### DISCUSSION THREAD

**Discussion by HuakunShen** [tauri-apps/tauri#11418] [Q&A]
I am thinking about embedding a encryption key using `include_bytes`, but without obfuscation the content could be easily read from binary.

**Comment by FabianLars** [comment]
Your frontend files will be brotli compressed (not really obfuscation but close enough to be mentioned), nothing else. The rust side will not be touched by Tauri.
You can use crates like litcrypt though. idk if that crate in particular is good, just the only one i can remember the name of.

**Reply by HuakunShen** [reply]
Thanks I will take a look. 
I tried inspecting tauri generated binary with strings command and could easily find the secret I embedded in it.
